setup.py

```diff
@@ -51,4 +51,7 @@
     install_requires=[
         'setuptools >= 0.6b1',
```

Comment:
setup.py will need to be modified to distribute the package rather than the module. It may be as simple as replacing `modules=` with `packages=`. Test `python setup.py sdist` and installing the subsequent sdist.